diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -313,7 +356,7 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule("newlines", conf
                 return
             }
         }
-        if (numEntries > configuration.maxParametersInOneLine) {
+        if (numEntries > configuration.maxParametersInOneLine || (node.text.length - 2) > configuration.maxLengthOneLine) {
```

Comment:
I suggest to remove length checks from this rule after all. I believe `LineLegth` rule already has some similar logic for arguments litsts.